src/PythonMigrationViewExtension/MigrationAssistant/reindent.py

```diff
@@ -0,0 +1,182 @@
+#! /usr/bin/env python3
+
+# Based on a script by Tim Peters, 03 October 2000.
```

Comment:
what is the license for this code?
should it be added to license / about etc?

Reply:
Not sure. I got the original script from the Python repo, so I guess it's the same license.